client/termdb/TermTypeSearch.ts

```diff
@@ -14,7 +14,7 @@ The tree target is used to determine the allowed term types.
 
 const useCases = {
 	matrix: [TermTypeGroups.DICTIONARY_VARIABLES, TermTypeGroups.MUTATION_CNV_FUSION],
-	filter: [TermTypeGroups.DICTIONARY_VARIABLES, TermTypeGroups.MUTATION_CNV_FUSION],
+	filter: [TermTypeGroups.DICTIONARY_VARIABLES, TermTypeGroups.MUTATION_CNV_FUSION, TermTypeGroups.GENE_EXPRESSION],
```

Comment:
please remove geneExp from filter, and do it in next branch where client/filter/tvs.geneExpression.js will be created. right now breaks

Reply:
I just did. It was pushed by mistake. Please delete the branch and pull again